app/tests/workstations_tests/test_permissions.py

```diff
@@ -262,3 +264,40 @@ def test_session_api_permissions(client, two_workstation_sets, viewname):
                 assert response.json()["count"] == 0
         else:
             assert response.status_code == test[1]
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("method", ["put", "patch"])
+def test_session_api_patch_permissions(client, two_workstation_sets, method):
+    tests = (
```

Comment:
Why not add these different tests also as parametrized tests?

Reply:
Because we need to parametrise it with a fixture, as far as I can tell this is not possible yet (there is a proposal here: https://docs.pytest.org/en/latest/proposals/parametrize_with_fixtures.html)